libr/core/cconfig.c

```diff
@@ -502,17 +502,14 @@ static int cb_asmarch(void *user, void *data) {
 
 	// set a default endianness
 	int bigbin = r_bin_is_big_endian (core->bin);
-	if (bigbin == -1 /* error: no endianness detected in binary */) {
-		// try to set RAsm to LE
-		r_asm_set_big_endian (core->assembler, false);
-		// set endian of display to LE
-		core->print->big_endian = false;
-	} else {
-		// try to set endian of RAsm to match binary
-		r_asm_set_big_endian (core->assembler, bigbin);
-		// set endian of display to match binary
-		core->print->big_endian = bigbin;
-	}
+	if (bigbin == -1 /* error: no endianness detected in binary */)
```

Comment:
add braces around the body

Reply:
why? according to https://github.com/radare/radare2/blob/master/CONTRIBUTING.md
"Always use braces for if and while, except when the expressions are very simple and they can fit in a one-line."
Are there any exception of this rule?

Reply:
then that file is wrong, should review it then :P